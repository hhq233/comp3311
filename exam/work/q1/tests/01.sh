psql beer -c 'select * from q1;'
