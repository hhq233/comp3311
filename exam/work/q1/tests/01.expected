             beer              |           brewery            | rating 
-------------------------------+------------------------------+--------
 Carlton Draught               | Carlton and United Breweries |      1
 Cero                          | Grupo Modelo - Corona        |      1
 Maple Banana Split Your Pants | Urban Alley Brewery          |      1
(3 rows)
